packages/plugin-tee/src/providers/deriveKeyProvider.ts

```diff
@@ -107,15 +165,17 @@ const deriveKeyProvider: Provider = {
                     runtime.getSetting("WALLET_SECRET_SALT") || "secret_salt";
                 const solanaKeypair = await provider.deriveEd25519Keypair(
                     "/",
-                    secretSalt
+                    secretSalt,
+                    agentId
                 );
                 const evmKeypair = await provider.deriveEcdsaKeypair(
                     "/",
-                    secretSalt
+                    secretSalt,
+                    agentId
                 );
                 return JSON.stringify({
-                    solana: solanaKeypair.publicKey,
-                    evm: evmKeypair.address,
+                    solana: solanaKeypair.keypair.publicKey,
+                    evm: evmKeypair.keypair.address,
```

Comment:
This return makes the `return keypair;` on L185 unreachable and also `let keypair: Keypair;` on L162 useless.

Reply:
Fixed, I was missing linter in my plugin. Added it to avoid this next time.